### DISCUSSION TITLE
conda install pygeos error

### DISCUSSION THREAD

**Discussion by emaildipen** [geopandas/geopandas#2744] [Installation]
Everytime I tried to install the following message shows up
The following packages are causing the inconsistency:

  - conda-forge/noarch::earthengine-api==0.1.249=pyh44b312d_0
  - conda-forge/noarch::geeadd==0.5.4=pyhd8ed1ab_0
  - conda-forge/noarch::geemap==0.7.9=pyh9f0ad1d_0
  - conda-forge/noarch::google-api-core==1.26.2=pyhd8ed1ab_0
  - conda-forge/noarch::google-api-python-client==2.0.2=pyhd8ed1ab_0
  - conda-forge/noarch::google-auth==1.28.0=pyh44b312d_0
  - conda-forge/noarch::google-auth-httplib2==0.0.4=pyh9f0ad1d_0
  - conda-forge/noarch::google-cloud-core==1.5.0=pyhd3deb0d_0
  - conda-forge/noarch::google-cloud-storage==1.28.1=pyh9f0ad1d_0
  - conda-forge/noarch::pyasn1-modules==0.2.7=py_0
  - conda-forge/noarch::rsa==4.7.2=pyh44b312d_0

**Comment by ccalvocm** [comment]
Greetings @emaildipen 

From Anaconda Prompt enter:

`conda create -n geopandas_venv -c conda-forge python=3.9 geopandas
`
`conda activate geopandas_venv`
`python`
`import geopandas`
